src/commands/capabilities/translations/update.ts

```diff
@@ -0,0 +1,48 @@
+import { type ArgumentsCamelCase, type Argv, type CommandModule } from 'yargs'
+
+import { type CapabilityLocalization } from '@smartthings/core-sdk'
+import { apiOrganizationCommand, apiOrganizationCommandBuilder, type APIOrganizationCommandFlags } from '../../../lib/command/api-organization-command.js'
+import { inputAndOutputItem, inputAndOutputItemBuilder, type InputAndOutputItemFlags } from '../../../lib/command/input-and-output-item.js'
+import { capabilityIdBuilder, type CapabilityIdInputFlags } from '../../../lib/command/capability-flags.js'
+import { apiDocsURL } from '../../../lib/command/api-command.js'
+import { chooseCapability } from '../../../lib/command/util/capabilities-choose.js'
+import { buildTableOutput } from '../../../lib/command/util/capabilities-translations-table.js'
+
+
+export type CommandArgs =
+	& APIOrganizationCommandFlags
+	& CapabilityIdInputFlags
+	& InputAndOutputItemFlags
+
+const command = 'capabilities:translations:update [id]'
+
+const describe = 'update a capability translation'
+
+const builder = (yargs: Argv): Argv<CommandArgs> =>
+	inputAndOutputItemBuilder(capabilityIdBuilder(apiOrganizationCommandBuilder(yargs)))
+		.positional('id', { describe: 'device preference id', type: 'string' })
```

Comment:
Capability rather than device preference?

Reply:
Oops, fixed by deleting that unnecessary line. The `id` positional is already taken care of by the call to `capabilityIdBuilder` on the preceding line.